content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md

```diff
@@ -0,0 +1,45 @@
+---
+weight: 40
+title: Release 10.18.0
+layout: bundle
+---
+
+Cumulocity IoT DataHub Release 10.18 includes the following improvements, limitations, and known issues:
+
+### New version of internal query engine
+
+Cumulocity IoT DataHub now uses version 24 of Dremio as query engine. The new version comes along with various features and improvements like a revamped user interface, simplified interaction with PowerBI, improved query performance.
+
+### Efficient ODBC connectivity
+
+The ODBC driver required for running queries against Dremio now relies on Apache Arrow, whose columnar memory format allows for a more efficient ODBC-based query execution.
+
+### Enforcement of single data types
+
+Columns having multiple data types, so-called mixed types, can no more occur. During the offloading process, Cumulocity IoT DataHub takes care that the offloaded data has one type per column. If incoming data might lead to multiple types, either the associated offloading pipeline will be stopped or the resulting schema is automatically evolved.
+
+### Improved handling of additional columns
+
+The UI simplifies duplicating an additional column and changing the data type of the column.
+
+### Limitations
+
+|<div style="width:250px">Description</div>
+|:---
+|Mixed usage of uppercase and lowercase characters for attribute names in the documents is not supported.|
```

Comment:
This is not the case anymore. Dremio now can read both lowercase and uppercase properties from MongoDB. I.e., myprop and myProp would now both be read (as myprop). The only remaining limitation is that if BOTH attributes are set for the same document, only one of the values would be visible.